### ISSUE TITLE
Drawing rois - interface is really slow, especially for the first scan... 

### ISSUE THREAD

**Issue by tomelse** [tomelse/patato#9]

**Comment by jgroehl** [comment]
Could this be an effect of the lazy loading you have added as a feature?
If that is the case, "touching" all slices before going into the GUI might do the trick :D

**Comment by tomelse** [comment]
Yes I think you're right. I will see how the H5Py library works and see if there's a quick fix. Something I've wanted to fix for a while though!
